src/main/webapp/app/shared/metis/postings-thread/postings-thread.component.html

```diff
@@ -13,6 +13,19 @@
                         (sortedAnswerPosts.length === 0 ? 'artemisApp.metis.startDiscussion' : 'artemisApp.metis.answerNow') | artemisTranslate
                     }}</span>
                 </button>
+                <!-- expand answers -->
+                <button *ngIf="!showAnswers && sortedAnswerPosts.length > 0" class="btn btn-sm btn-outline-secondary mt-2" (click)="showAnswers = true">
+                    <fa-icon [icon]="'comments'"></fa-icon>
+                    <span class="d-none d-md-inline">
+                        {{ 'artemisApp.metis.showAnswers' | artemisTranslate: { number: sortedAnswerPosts.length } }}
+                        {{ (sortedAnswerPosts.length === 1 ? 'artemisApp.metis.answerSingular' : 'artemisApp.metis.answerPlural') | artemisTranslate }}</span
```

Comment:
```suggestion
                        {{ 'artemisApp.metis.showAnswer' + (sortedAnswerPosts.length === 1 ? 'singular' : 'plural') | artemisTranslate: { number: sortedAnswerPosts.length} }}</span
```

Reply:
I changed the logic here, it's easier to read now